index.js

```diff
@@ -280,6 +283,14 @@ function Url(address, location, parser) {
     url.pathname = resolve(url.pathname, location.pathname);
   }
 
+  //
+  // Default to a / for pathname if none exists. This normalizes the URL
+  // to always have a /
+  //
+  if (url.pathname.charAt(0) !== '/' && url.hostname) {
+    url.pathname = '/' + url.pathname;
```

Comment:
This cause break change, see below demo code:
```
// assume this code exec in page http://cone-cf8b5c0e.app-dev.alipay.net/cone/strategy
const { pathname} = url('/cone/operate');
```
This PR cause pathname change from 'cone/operate' to '/cone/cone/operate'

debugger snapshot:
input: 
![image](https://user-images.githubusercontent.com/7971415/108344002-8ff5d880-7217-11eb-823e-84dc73cec269.png)

output:
![image](https://user-images.githubusercontent.com/7971415/108344060-a308a880-7217-11eb-8ed1-e0855e8f121a.png)

i don't know of pass '/cone/operate' is a valid argument,  if no, i think throw error is a better way

Reply:
Could you create an issue about this so we can track it? We do known issue with relative paths atm see #200